app/build.gradle.kts

```diff
@@ -15,18 +15,18 @@
  */
 plugins {
     id("com.android.application")
-    id("com.google.devtools.ksp") version "1.8.21-1.0.11"
+    id("com.google.devtools.ksp") version "1.9.0-1.0.13"
     id("org.jetbrains.kotlin.android")
 }
 
 android {
     namespace = "com.example.busschedule"
-    compileSdk = 33
```

Comment:
Since Giraffe template code uses SDK 33, Murat suggested setting target SDK to 33 for now.

Reply:
This requires lowering other version as well. 
Is there a specific reason to do this?